pages/iam/how-to/set-up-identity-federation.mdx

```diff
@@ -0,0 +1,127 @@
+dates:
+  validation: 2025-09-21
+  posted: 2025-08-21
+---
+import Requirements from '@macros/iam/requirements.mdx'
+
+Scaleway supports Identity Federation to provide your teams with secure access to their accounts via Single Sign-On (SSO). Depending on your organization’s requirements, you can use either built-in OAuth2 providers or configure SAML for centralized identity management.
+
+| Feature | **OAuth2** | **SAML** |
+|--------|-------------------------------|--------|
+| **Availability** | Enabled by default for all organizations | Available, but requires setup |
+| **Supported Providers** | Google, GitHub | Any SAML-compatible Identity Provider |
+| **Setup Required** | No | Yes — must be configured by an IAM admin |
+| **User Access** | Any Scaleway member whose email is verified with Google or GitHub | Only users explicitly defined in the Identity Provider |
+| **Centralized Management** | No | Yes — manage users from your Identity Provider |
+
+<Message type="important">
+  Keep in mind that:
+  - OAuth2 logins are automatically disabled when SAML is configured. If SAML is not set up, members can continue to use Google or GitHub for SSO.
+  - SSO with SAML does not apply to an Organization's Owner. Owners can log in with SSO with OAuth2.
+</Message>
+
+Follow the steps below to set up Identity Federation for your Organization through SAML at Scaleway.
+
+<Requirements />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/iam/concepts/#owner) status or [IAM permissions](/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+- An Identity Provider (IdP) configured in your company, making sure it includes all users who need to access Scaleway. Some examples of IdPs:
+   - Okta
+   - OneLogin
+   - Microsoft Entra ID (prev. Azure AD)
+   - PingIdentity
+   - Google Workspace
+   - Keycloak
+
+## How to set up a SAML connection
+
+1. Go to your security settings.
+2. Scroll to the **Identity Federation** section.
+3. Click **Set up SSO**. A pop-up appears.
+4. Copy the URLs displayed in the pop-up.
+
+    The information in the first step are the URLs referring to Scaleway that will be requested by your Identity Provider to create a link between platforms. They are:
+        - The assertion consumer service (ACS) URL, and
+        - Scaleway's entity ID
+
+5. Click **Next**.
+6. Enter the requested URLs in their respective boxes.
+
+    This is the information referring to your Identity Provider that Scaleway needs to confirm the connection. It can be found in your IdP's configuration page. They are:
+        - **Single Sign-On URL** - This is the URL your members will be redirected to when logging in with SAML
+        - **The Identity Provider's Entity ID**
+7. Click **Confirm**.
+8. Enter the signing certificate generated by your Identity Provider in the box.
+    <Message type="important">
+    You certificate entry must start with:
```

Comment:
We should probably more stated that we accept only "base64-encoded certificates". We should also state that the certificat must have an expiration date.